src/northbridge/amd/amdfam10/northbridge.c

```diff
@@ -536,6 +537,16 @@ static void amdfam10_create_vga_resource(struct device *dev, unsigned int nodeid
 
 	printk(BIOS_DEBUG, "VGA: %s (aka node %d) link %d has VGA device\n", dev_path(dev), nodeid, link->link_num);
 	set_vga_enable_reg(nodeid, link->link_num);
+
+
+	/* Redirect VGA memory access to MMIO This signals the Family 10h
+	 * resource parser to add a new MMIO mapping to the Range 11 MMIO
+	 * control registers (starting at F1x1B8), and also reserves the
+	 * resource in the E820 map.
+	 */
+	res = amdfam10_assign_new_mmio_res(0xa0000, 0x20000);
```

Comment:
`mmio_resource()` which is already present in `amdfam10_domain_read_resources()` should do the same for E820 map (but this would be better place for that call), and adding it to MMIO control registers shouldn't be necessary because `set_vga_enable_reg()` enables this routing in different register. Also, the comment above does not match the code, it doesn't use `F1x1B8`.

Reply:
Removed.